maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -45,7 +45,10 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
-import static org.jboss.pnc.mavenrepositorymanager.MavenRepositoryConstants.*;
+import static org.jboss.pnc.mavenrepositorymanager.MavenRepositoryConstants.DRIVER_ID;
+import static org.jboss.pnc.mavenrepositorymanager.MavenRepositoryConstants.PUBLIC_GROUP_ID;
+import static org.jboss.pnc.mavenrepositorymanager.MavenRepositoryConstants.SHARED_IMPORTS_ID;
+import static org.jboss.pnc.mavenrepositorymanager.MavenRepositoryConstants.UNTESTED_BUILDS_GROUP;
```

Comment:
I would leave hardcoded DRIVER_ID, agree for other to move to config.
Actually my update to use static imports was not intentional.